src/main/java/frc/trigon/robot/subsystems/shooter/ShooterCommands.java

```diff
@@ -16,7 +16,23 @@ public static Command getDebuggingCommand() {
         );
     }
 
-    public static Command getReachTargetShootingVelocityFromShootingCalculations() {
+    public static Command getShootToAmpCommand() {
+        return new ExecuteEndCommand(
+                RobotContainer.SHOOTER::shootToAmp,
+                RobotContainer.SHOOTER::stop,
+                RobotContainer.SHOOTER
+        );
+    }
+
+    public static Command getShootCloseShotCommand() {
+        return new ExecuteEndCommand(
+                RobotContainer.SHOOTER::shootCloseShot,
+                RobotContainer.SHOOTER::stop,
+                RobotContainer.SHOOTER
+        );
+    }
```

Comment:
Honestly, instead of having these two, I would just call `getSetTargetVelocityCommand` with a constant velocity.